chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/MeaningfulUseAccurateAsOf.java

```diff
@@ -0,0 +1,97 @@
+package gov.healthit.chpl.domain;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+
+import gov.healthit.chpl.dto.MeaningfulUseAccurateAsOfDTO;
+
+public class MeaningfulUseAccurateAsOf implements Serializable {
+	private static final long serialVersionUID = -4803763243075068608L;
+	
+	private Long id;
+	private Long accurateAsOfDate;
+	private Boolean deleted;
+	private Long lastModifiedUser;
+	private Long creationDate;
+	private Long lastModifiedDate;
+	
+	private Set<String> warningMessages = new HashSet<String>();
```

Comment:
I left a comment down there, too. :) Only Certified Products and Surveillance have errorMessages/warningMessages since those are things that are uploaded and can have a long list of things wrong with them. In cases where it's a single error that's occurring the restful way to do it is just send an appropriate error code with a message describing what is wrong.